lib/ansible/executor/action_write_locks.py

```diff
@@ -36,8 +37,7 @@
 
     # These plugins are called directly by action plugins (not going through
     # a strategy).  We precreate them here as an optimization
-    mods = set(p['name'] for p in Facts.PKG_MGRS)
+    mods = set(C.PKG_MGR_MODULES)
```

Comment:
nothing in module_utils should have deps  outside of it, but its fine for stuff in ansible to depend on module_utils. That said, not sure what/why PKG_MGR is needed here or why we are using these locks in the first place